modern_treasury/objects/webhook_events/base.py

```diff
@@ -0,0 +1,55 @@
+from decimal import Decimal
+
+
+class MetaData:
```

Comment:
I would take these out, anything specific to EZ. We can extend this class on the Kemcho side. 
Since this is a public lib we would like it so non specific data for out company does not go into this repo.